src/plugins/github/fetchGithubRepo.js

```diff
@@ -45,101 +45,14 @@ export default function fetchGithubRepo(
     throw new Error(`Invalid token: ${token}`);
   }
 
-  const query = GraphQLQueries.stringify.body(
-    createQuery(),
-    GraphQLQueries.inlineLayout()
-  );
-  const variables = {repoOwner, repoName};
+  const query = stringify.body(createQuery(), inlineLayout());
+  const variables = createVariables(repoOwner, repoName);
```

Comment:
why do we want this createVariables function given that it's a pretty trivial one line expression?

Reply:
The variable names are now encapsulated by this module. Code outside the module doesn't know the variable names, doesn't need to, and shouldn't.

Reply:
Got it.